INSTALL.md

```diff
@@ -270,10 +270,13 @@ Creating the database
     By default, the password for the user musicbrainz should be "musicbrainz",
     as stated in lib/DBDefs.pm. You can change it with `psql`:
 
-        postgres=# ALTER USER musicbrainz UNENCRYPTED PASSWORD 'musicbrainz'
+        postgres=# ALTER USER musicbrainz UNENCRYPTED PASSWORD 'musicbrainz';
+
+    For PostgreSQL 10 or later, you will need to leave off the `UNENCRYPTED`.
```

Comment:
Huh, I can't see any reason to specify `UNENCRYPTED` in 9.5 either. According to the docs, that indicated whether the given password was to be encrypted in the system catalog (not whether the given password was already in encrypted format). I'd just remove `UNENCRYPTED` from the original line.

Reply:
The 9.5 docs do not quite make it clear whether leaving it off would assume/require the value to be an md5 hash. I could switch it around and say that 9.x might require `UNENCRYPTED`.

Would it also make sense to change the main line to use `CREATE`, given this is mostly a set-up-from-scratch guide (mentioning `ALTER`In text afterwards)?

Reply:
The relevant docs for ENCRYPTED / UNENCRYPTED are actually found under [CREATE ROLE](https://www.postgresql.org/docs/9.5/sql-createrole.html) since those options are inherited from there:

> These key words control whether the password is stored encrypted in the system catalogs. (If neither is specified, the default behavior is determined by the configuration parameter password_encryption.) If the presented password string is already in MD5-encrypted format, then it is stored encrypted as-is, regardless of whether ENCRYPTED or UNENCRYPTED is specified (since the system cannot decrypt the specified encrypted password string). This allows reloading of encrypted passwords during dump/restore.

So I don't see any reason to specify `UNENCRYPTED` for 9.5.

Re: changing it to `CREATE`, this is actually kind of weird since admin/InitDb.pl creates the musicbrainz user if it doesn't exist (setting the password to the one specified in DBDefs). So maybe we should just say to make sure the desired password in specified for `READWRITE` in DBDefs before running InitDb.pl. I'm not sure we really need to document how to change the password once the user is already created.

https://github.com/metabrainz/musicbrainz-server/blob/396d0ca13a917d1b7fddd8c1a52e18bb12c15f22/admin/InitDb.pl#L200-L211

Reply:
OK, even easier, I'll drop that paragraph. I always manually created the user before running InitDB, because of where this ALTER was documented.